src/lib.rs

```diff
@@ -118,231 +422,25 @@ impl AggregatedQueries {
                 aggregated_query_text.push_str("\n\n");
             }
         }
-        let query = Query::new(context.language, &aggregated_query_text).unwrap();
+        let query = Query::new(config.language.language(), &aggregated_query_text).unwrap();
         assert!(query.pattern_count() == pattern_index_lookup.len());
         Self {
             pattern_index_lookup,
             query,
             query_text: aggregated_query_text,
+            resolved_rules,
         }
     }
-}
-
-fn get_rules() -> Vec<Rule> {
-    vec![
-        no_default_default_rule(),
-        no_lazy_static_rule(),
-        prefer_impl_param_rule(),
-    ]
-}
-
-fn no_default_default_rule() -> Rule {
```

Comment:
Even though these are just temporarily hard-coded I didn't like having them here in `src/lib.rs` so moved them into separate files